app/templates/account/profile.html

```diff
@@ -6,6 +6,19 @@
     <div>{{ user.email }}</div>
     <div>{{ user.role.name }}</div>
     <div>{{ user.birthday }}</div>
+    <div>
+        {% if user.tags %}
+            {% for affiliation in user.tags %}
+            {% if affiliation.type == "AffiliationTag" %}
```

Comment:
I'm going to be a stickler and ask that the stuff in the for loop (12-16) gets indented one to the right please